spec/features/analyses/gwas/creation_spec.rb

```diff
@@ -59,6 +66,42 @@
     end
   end
 
+  context "with plant trial selected" do
+    let!(:trait_descriptors) { create_list(:trait_descriptor, 5) }
+    let!(:plant_scoring_units) {
+      1.upto(100).map { |idx|
+        create(:plant_scoring_unit, plant_trial: plant_trial, scoring_unit_name: "plant#{idx}")
+      }
+    }
+    let!(:trait_scores) {
+      plant_scoring_units.map.with_index { |psu, idx|
+        create(:trait_score, plant_scoring_unit: psu, score_value: idx, trait_descriptor: trait_descriptors.sample)
+      }
+    }
+
+    scenario "is successful", js: true do
+      click_link("Start!")
+      fill_in("Analysis name", with: "Some interesting name")
+
+      select(plant_trial.plant_trial_name, from: "analysis_plant_trial_id")
+
+      attach_file("genotype-data-file", fixture_file_path("gwas-genotypes.csv"))
+      attach_file("map-data-file", fixture_file_path("gwas-map.csv"))
+
+      sleep(1)
```

Comment:
No, it is needed for the upload to be performed, but I found more explicit way to check that.